mayday.go

```diff
@@ -62,6 +62,7 @@ func openFile(f File) (*file.MaydayFile, error) {
 
 func main() {
 	pflag.BoolP("danger", "d", false, "collect potentially sensitive information (ex, container logs)")
+	pflag.StringP("output", "o", "", "output file (default: /tmp/mayday{current time}.tar.gz)")
```

Comment:
One thing... let's add in the hostname by default too.  That way when we get to the stage of them being uploaded to a remote endpoint they can be differentiated.  Now that the name can be included it can be overridden if someone doesn't like that.